src/metrics/logic.py

```diff
@@ -38,14 +39,39 @@ def notify(obj):
     else:
         events.notify(obj)
 
+'''
+def recently_updated_articles():
+    "all articles whose associated metrics/citations have been updated in the last hour"
+    since = utils.utcnow() - timedelta(hours=2)
+    recent_citations = Q(citation__datetime_record_updated__gte=since)
+    recent_views_downloads = Q(metric__datetime_record_updated__gte=since)
+    return models.Article.objects.filter(recent_citations | recent_views_downloads).distinct()
+'''
+
+def recently_updated_citations():
+    "all articles whose associated metrics/citations have been updated in the last hour"
```

Comment:
good catch. yes, the total time to import all metrics is just over 40mins last I checked. another (more robust approach) would be to start a timer, do the imports, then pass the elapsed time to the `recently_updated_*` fns. In fact, I might do just that.